### DISCUSSION TITLE
--max_line_count is unrecognized

### DISCUSSION THREAD

**Discussion by MindSetFPS** [openai/whisper#1301] [Q&A]
pretty much what the title says. the option throws the following error:

whisper: error: unrecognized arguments: --max_line_count

whisper version: 20230314

**Comment by ryanheise** [comment]
Version 20230314 was released on March 14th, while `--max_line_count` added in April, so to run it you'll need to install the latest version from the git repository. The README provides instructions on how to install the latest version from git.
